src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,12 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    &:hover {
+      transform: scale(1.2, 1.2);
+      color: #34568b;
+      transition: transform $smooth-on-hover, color $smooth-on-hover;
```

Comment:
you should add transition on whole class rather than on hover, so transition works both ways